src/components/Header/BrowseBtn.js

```diff
@@ -26,9 +21,8 @@ const MobileLinkContainer = styled.div`
   }
 
   @media only screen
-      and (max-height: 767px)
-      and (orientation: portrait) {
-        height:7em;
+      and (max-width: 1024px) {
```

Comment:
BrowseBtn has been replaced with ../Footer/FooterButton.js if you remove this will add directly to master